src/PaymentGateways/Gateways/Offline/resources/formBuilder/withOfflineInspectorControls.tsx

```diff
@@ -87,7 +87,6 @@ function OfflineInspectorControls({
                                 'give'
                             )}
                             heading={__('Donation Instructions', 'give')}
-                            buttonCaption={__('Edit', 'give')}
```

Comment:
Was this intentional?

Reply:
Yes, @jdghinson asked that it follows the same standard as the other Popover controls. By removing it here, it will default to the PopoverEdit Icon.